l10n_it_statutory_financial_statements/readme/USAGE.rst

```diff
@@ -0,0 +1,15 @@
+**Italiano**
+
+Eseguire
+
+Fatturazione > Rendicontazione > Bilancino
+
+Selezionare "Bilancio civilistico"
```

Comment:
La stringa inglese risulta "Civil code statutory financial statements" (forse un refuso? vedi https://github.com/OCA/l10n-italy/pull/1264#discussion_r291841925). In caso andrebbe semplificata in "Statutory financial statements"

Se invece vuoi mantenere la forma completa, in italiano potrebbe essere un cosa tipo "Bilancio di esercizio (codice civile)" o "Bilancio di esercizio - Codice civile"  (per evitare il doppione civilistico/civile)